src/OpenTelemetry/Logs/BatchExportLogRecordProcessorOptions.cs

```diff
@@ -27,18 +27,6 @@ namespace OpenTelemetry.Logs;
 /// OTEL_BLRP_SCHEDULE_DELAY environment variables are parsed during object
 /// construction.
 /// </summary>
-/// <remarks>
-/// Notes:
-/// <list type="bullet">
-/// <item>The constructor throws <see cref="FormatException"/> if it fails to
```

Comment:
For anyone curious the FormatException logic was changed a while back so this comment was lying: https://github.com/open-telemetry/opentelemetry-dotnet/pull/4095